packages/core-data/src/index.js

```diff
@@ -31,27 +35,17 @@ const entitiesConfig = [
 
 const entitySelectors = entitiesConfig.reduce( ( result, entity ) => {
 	const { kind, name, plural } = entity;
-	result[ getMethodName( kind, name ) ] = ( state, key, query ) =>
-		selectors.getEntityRecord( state, kind, name, key, query );
-
-	if ( plural ) {
-		result[ getMethodName( kind, plural, 'get' ) ] = ( state, query ) =>
-			selectors.getEntityRecords( state, kind, name, query );
-	}
-	return result;
-}, {} );
-
-const entityResolvers = entitiesConfig.reduce( ( result, entity ) => {
-	const { kind, name, plural } = entity;
-	result[ getMethodName( kind, name ) ] = ( key, query ) =>
-		resolvers.getEntityRecord( kind, name, key, query );
+	result[ getMethodName( kind, name ) ] = createRegistrySelector(
+		( select ) => ( state, key, query ) =>
+			select( STORE_NAME ).getEntityRecord( kind, name, key, query )
+	);
```

Comment:
Here `createRegistrySelector` is not used to select from _another_ store, but to select from the same store, we just want to call the selectors that are already bound to the store, with all the resolver-calling wrappers, instead of the unbound raw functions.

That's fine, but I think we should eventually come up with some new API for this.

Reply:
What do you have in mind? I was thinking of creating something to pass aliases for selector/resolvers when creating the store, and handle them in the store

Reply:
Yep, we do the same thing to trigger resolvers from other normal selectors in the same store. A better solution is needed there too.